<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentTrackPopover</class>
 <widget class="QWidget" name="CurrentTrackPopover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="artLabel">
       <property name="text">
        <string notr="true">Art</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="metadataLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="skipBackButton">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset theme="media-skip-backward"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="playButton">
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset theme="media-playback-start"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="skipNextButton">
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset theme="media-skip-forward"/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="metadataEntryLayout"/>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
